README.md

```diff
@@ -250,6 +250,38 @@ Please click [Cross Compilation](docs/setup/cross-compilation.md) for the instru
 
 ### Run Cloud
 
+#### Run as Kubernetes deployment
+
+This method will guide you to deploy the cloud part into a k8s cluster,
+so you need to login to the k8s master node (or where else if you can
+operate the cluster with `kubectl`).
+
+The manifests and scripts in `github.com/kubeedge/kubeedge/build/cloud`
+will be used, so place these files to somewhere you can kubectl with.
+
+Then, first, we need to generate the tls certs (and CA if we don't have
+one) for CloudHub to host websocket server. It then will give us
```

Comment:
for CloudHub to host websocket server.   --> what does this mean ? can you clarify a little more ?

Reply:
Internal cloud hub module, there is a websocket server object, to start the server for accepting edge (client connection), a config object which contains tls certs is need.
It means cloud hub module need tls certs. I will ask someone good at English to help me clarify these words.